app/Http/Controllers/PostController.php

```diff
@@ -27,26 +27,34 @@ public function __construct()
     public function show($id)
     {
         $data = []; //to be sent to the view
-
-        $post = Post::findOrFail($id);
+        $post = Post::with("comments")->find($id);
```

Comment:
Use findOrFail

Reply:
I this case I need it. You weren't in class. He said that this way it is more optimal.

Reply:
Why don't you explain why it is better this way?